engine/api/workflow/dao.go

```diff
@@ -1183,6 +1183,9 @@ func checkHooks(db gorp.SqlExecutor, w *sdk.Workflow, n *sdk.Node) error {
 			w.HookModels[hm.ID] = *hm
 			h.HookModelID = hm.ID
 		}
+		if h.HookModelName == sdk.RepositoryWebHookModelName && (n.Context == nil || n.Context.ApplicationID == 0) {
```

Comment:
there is a risk that we will be unable to load some existing workflows, we have to make a migration is any